airgun/entities/computeresource.py

```diff
@@ -80,7 +100,7 @@ def step(self, *args, **kwargs):
 
 @navigator.register(ComputeResourceEntity, 'Edit')
 class EditExistingComputeResource(NavigateStep):
-    VIEW = ResourceProviderEditView
+    VIEW = ResourceProviderCreateView
```

Comment:
CreateView for Edit step? Sounds wrong to me. You should have separate Create/Edit views, as entity does have breadcrumb widget. If views are the same and the only difference is breadcrumb values - just inherit one view from another and override `is_displayed` property. Take a look at architecture/computeprofile/other views for example on how it's implemented.